source/crud/tests/README.rst

```diff
@@ -19,20 +19,21 @@ version requirements as noted by the ``runOn`` section, if provided.
 Subdirectories for Test Formats
 -------------------------------
 
-This document describes a current test format, which should be used for any new
-CRUD tests. Additionally, it refers to a "legacy" format, which dates back to
-the initial version of the CRUD specification. Until such time that all original
-tests have been ported to the current format, tests in each format will be
-grouped in their own subdirectory:
+This document describes two legacy formats for CRUD tests: legacy-v1, which dates back
+to the first version of the CRUD specification, and legacy-v2, which was an update to
+the initial format. New CRUD tests should be written in the `unified test format <../../../../unified-test-format/unified-test-format.rst>`_
+and placed under ``unified/``. Until such time that all original tests have been ported
+to the unified test format, tests in each format will be grouped in their own subdirectory:
 
-- ``v1/``: Legacy format tests
-- ``v2/``: Current format tests
+- ``v1/``: Legacy-v1 format tests
+- ``v2/``: Legacy-v2 format tests
+- ``unified/``: Tests using the `unified test format <../../../../unified-test-format/unified-test-format.rst>`_
 
 Since some drivers may not have a unified test runner capable of executing tests
-in both formats, segregating tests in this manner will make it easier for
+in all three formats, segregating tests in this manner will make it easier for
 drivers to sync and feed test files to different test runners.
 
-Test Format
+Legacy-v2 Test Format
```

Comment:
Apologies for missing this. I'm to sure why this wasn't caught by rstcheck, but I'm not very familiar with the tool. My only interaction was to migrate it to GH actions. From what I can see, the tool did not fail, so we may want to revisit our RST linting (@benjirewis feel free to suggest an alternative if you found that this catches errors more reliably).

Reply:
@alcaeus  `rstcheck` works great! The reason it wasn't caught is because it's [called](https://github.com/mongodb/specifications/blob/master/.github/workflows/rstcheck.yml#L25) with the `--report error` flag on the GH action. "Title underline too short" is a warning, not an error. So, we'd have to change the action to use `--report warning` to catch it. Not sure if you want all the other warnings, though...

Reply:
Ah, I see. I'll take a look through the warnings and see what we can do 👍